index.html

```diff
@@ -33,11 +33,175 @@ <h2 id="id-my-name">Marc Raphael</h2>
+              <span class="class-project-span">HTML/CSS</span>
+              <span class="class-project-span">Ruby on rails</span>
+              <span class="class-project-span">Javascript</span>
+            </div>
+            <a href="#" target="_parent" class="class-project-link">
+              <span class="class-project-button">See this Project</span>
+              <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+            </a>
+          </div>
+        </div>
+
+        <!--project five-->
+        <div class="class-project">
+          <img src="./images/Rectangle21(5).png" alt="project image" class="class-project-image">
+          <div class="class-project-description">
+            <div class="class-project-languages">
+              <p class="class-project-name">Project name goes here</p>
+              <span class="class-project-span">HTML/CSS</span>
+              <span class="class-project-span">Ruby on rails</span>
+              <span class="class-project-span">Javascript</span>
+            </div>
+            <a href="#" target="_parent" class="class-project-link">
+              <span class="class-project-button">See this Project</span>
+              <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+            </a>
+          </div>
+        </div>
+        
+        <!--project six-->
+        <div class="class-project">
+          <img src="./images/Rectangle21(6).png" alt="project image" class="class-project-image">
+          <div class="class-project-description">
+            <div class="class-project-languages">
+              <p class="class-project-name">Project name goes here</p>
+              <span class="class-project-span">HTML/CSS</span>
+              <span class="class-project-span">Ruby on rails</span>
+              <span class="class-project-span">Javascript</span>
+            </div>
+            <a href="#" target="_parent" class="class-project-link">
+              <span class="class-project-button">See this Project</span>
+              <img src="./images/Union.png" alt="union pic" class="class-union-pic">
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <!--The section about my self-->
+      <section id="id-section-about-my-self">
+
+        <!--Introduction to the the project list-->
+        <h1 id="id-about-title">About</h1>
+
+        <!--litle description about me-->
+        <p id="id-about-description">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, 
+          sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+          Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+        </p>
+
+        <!--Button to the CV-->
+        <a href="#" target="_parent" id="id-about-cv">
+          <span id="id-about-span">Get my CV</span>
+          <img src="./images/ic_download.svg" alt="union pic" id="id-about-pic">
+        </a>
```

Comment:
- [ ] This should also use a header `hx` tag.
**_Refer to my comment above._**
- [ ] Check the templates for the design used in it, especially `font styles`. The `font-weight` used here is a lot heavier than that of the original

**Original** | **Your clone**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/120813184-84969680-c502-11eb-9e04-0f9e6ba31dd6.png) | ![image](https://user-images.githubusercontent.com/49621279/120813250-911aef00-c502-11eb-86cf-557a8f5ed3b1.png)

Reply:
- [ ] OPTIONAL: There is still some difference in the font-weight but I leave it as an optional improvement as it still looks ok for me.